priam/src/main/java/com/netflix/priam/tuner/StandardTuner.java

```diff
@@ -201,12 +214,10 @@ protected void configureCommitLogBackups() throws IOException {
         props.put("restore_directories", config.getCommitLogBackupRestoreFromDirs());
         props.put("restore_point_in_time", config.getCommitLogBackupRestorePointInTime());
 
-        FileOutputStream fos = null;
-        try {
-            fos = new FileOutputStream(new File(config.getCommitLogBackupPropsFile()));
+        try (FileOutputStream fos = new FileOutputStream(new File(config.getCommitLogBackupPropsFile()))) {
             props.store(fos, "cassandra commit log archive props, as written by priam");
-        } finally {
-            IOUtils.closeQuietly(fos);
+        } catch (IOException e) {
+            logger.error("Could not store commitlog_archiving.properties", e);
```

Comment:
Why are we eating exception here? Earlier priam would have been in restart loop but now it will just proceed with next steps?

Reply:
Ok, adding back the throw.